### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by madebyankur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,14 +97,15 @@ Finally, it will save environment variables for your tenant in the application d
 
 1. Run the following command:
 
-**Important: this will create and update entities in your Auth0 tenant – it is best to use a fresh/new tenant when bootstrapping. You can sign up for a free tenant at https://auth0.com/signup.
+**Important: this will create and update entities in your Auth0 tenant – it is best to use a fresh/new tenant when bootstrapping. You can sign up for a free tenant at [https://auth0.com/signup?utm_source=github&utm_medium=thirdparty&utm_campaign=saastart](https://auth0.com/signup).
```
Comment (matthewpereira) [inline README.md:100]: This is backward - it'll display the full URL but clicked will direct users to the one without a query string. Could you reverse the [] content and the () content?

File: README.md [hunk 2]
```diff
@@ -96,14 +97,15 @@ Finally, it will save environment variables for your tenant in the application d
...
-   ```
-    Behind the scenes, the bootstrap script will use the Auth0 CLI to provision the resources required for this sample application: creating the appropriate clients, roles, actions, templates, and MFA factors, and then saving environment variables for your tenant.
+```shell
+npm run auth0:bootstrap
+```
+
+Behind the scenes, the bootstrap script will use the Auth0 CLI to provision the resources required for this sample application: creating the appropriate clients, roles, actions, templates, and MFA factors, and then saving environment variables for your tenant.
 
-   Once the script has successfully completed, a `.env.local` file containing the environment variables will be written to the root of your project directory.
+Once the script has successfully completed, a `.env.local` file containing the environment variables will be written to the root of your project directory.
```
Comment (matthewpereira) [inline README.md:108]: What's the motivation for un-indenting this material? I think it previously 'nested' under `1. Run the following command:` as a result of the indentation.

**Review by matthewpereira — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (madebyankur) [reply README.md:100]: Nice catch 🙇‍♂️

File: README.md (hunk 2 above)
Comment (madebyankur) [reply README.md:106]: This wasn't formatted correctly as you can see on the `main` branch but I pushed some more updates - we don't need the single step here IMO.

File: README.md [hunk 3]
```diff
@@ -70,50 +71,51 @@ This project uses the Auth0 CLI to make setting up your tenant a lot easier, by
...
 1. Run the development server: `npm run dev`
-1. Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+2. Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
    > Note: If you're running the application on a different port, adjust the provided localhost URL accordingly.
 
-1. Start editing - for example, modify `app/page.tsx`. The browser will auto-update as you edit the file.
-> Note: If you're running the application on a different port, adjust the provided localhost URL accordingly.
+3. Start editing - for example, modify `app/page.tsx`. The browser will auto-update as you edit the file.
+   > Note: If you're running the application on a different port, adjust the provided localhost URL accordingly.
```
Comment (matthewpereira) [inline README.md:118]:
```suggestion

```

Comment (matthewpereira) [reply README.md:118]: This line is duplicated from 115 above.

**Review by matthewpereira — APPROVED** [review]

**Pull request merged**